content/php/concepts/string-functions/terms/htmlentities/htmlentities.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: 'htmlentities()'
+Description: 'Converts all applicable characters to HTML entities.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Design'
+  - 'Web Development'
+Tags:
+  - 'Strings'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`htmlentities()`** function converts characters to HTML entities. To convert HTML entities back to characters, use the **[html_entity_decode()](../html-entity-decode/html-entity-decode.md)** function.
```

Comment:
```suggestion
The **`htmlentities()`** function converts characters to HTML entities. To convert HTML entities back to characters, use the `html_entity_decode()` function.
```
Two things here: 1) we cannot link to an article that hasn't been merged unless it's in the same PR. 2) The link to the entry does not go to the GitHub repository, it would be linking to `https://www.codecademy.com/resources/docs/php/string-functions/html-entity-decode`